spec/acceptance/stub_spec.rb

```diff
@@ -59,6 +59,57 @@
   end
 end
 
+describe "a stubbed instance with a uuid primary key" do
+  it "builds a stubbed instance" do
+    using_model_with_uuid do
+      FactoryBot.define do
+        factory :model_with_uuid
+      end
+
+      model = FactoryBot.build_stubbed(:model_with_uuid)
+      expect(model).to be_truthy
+    end
+  end
+
+  it "behaves like a persisted record" do
+    using_model_with_uuid do
+      FactoryBot.define do
+        factory :model_with_uuid
+      end
+
+      model = FactoryBot.build_stubbed(:model_with_uuid)
+      expect(model).to be_persisted
+      expect(model).not_to be_new_record
+    end
+  end
+
+  it "has a uuid primary key" do
+    using_model_with_uuid do
+      FactoryBot.define do
+        factory :model_with_uuid
+      end
+
+      model = FactoryBot.build_stubbed(:model_with_uuid)
+      expect(model.id).to be_a(String)
+    end
+  end
+
+  def using_model_with_uuid
```

Comment:
It might be nice to update the existing helper to something more this, rather than adding a second very similar helper:

```rb
  def using_model(name, primary_key:)
    define_class(name, ActiveRecord::Base)

    connection = ActiveRecord::Base.connection
    begin
      clear_generated_table(name.tableize)
      connection.create_table(name.tableize, id: primary_key) do |t|
        t.column :updated_at, :datetime
      end

      yield
    ensure
      clear_generated_table(name.tableize)
    end
  end
```

Then we could call  `using_model("ModelWithoutPk", primary_key: false)` in the existing tests, and `using_model("ModelWithUuid", primary_key: :uuid)` in these new tests.

We'd probably have to wrap this whole file in an extra describe block to be able to share the helper method.